.github/workflows/docker-ghcr.yml

```diff
@@ -36,6 +35,14 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@v4
 
+      # Sets a new environment variable that can be accessed in later
+      # steps via ${{ env.ghcr-scope }}
+      - name: Get repository owner in lower-case
+        run: |
+          echo "ghcr-scope=${REPOSITORY_OWNER,,}" >> $GITHUB_ENV
```

Comment:
@aryanpingle What does this line do?

Reply:
It's a substitution command in bash that converts the variable to lowercase, [here's a link to a popular answer that I referred to](https://github.com/orgs/community/discussions/27086#discussioncomment-3254548).